metricsaggregator/src/test/resources/metrics-aggregator.config

```diff
@@ -0,0 +1,7 @@
+[dockstore]
+server-url: http://localhost:8080
+token: 08932ab0c9ae39a880905666902f8659633ae0232e94ba9f3d2094cb928397e7
```

Comment:
This is a dummy token: https://github.com/dockstore/dockstore/blob/60e4dedf3200c07fe2d9fb42778a98c00b0a99ca/dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java#L80

Reply:
Add this as a code comment
https://stackoverflow.com/questions/43217469/how-do-you-comment-out-lines-in-aws-cli-config-and-credentials-files